utopia-remix/app/routes/projects.tsx

```diff
@@ -70,76 +112,302 @@ const ProjectsPage = React.memo(() => {
     }
   }, [projectsFetcher.data])
 
+  const categories = [
+    { name: 'All My Projects', color: 'selected' },
+    // { name: 'Private', color: 'neutral' },
+    // { name: 'Public', color: 'neutral' },
+    // { name: 'Shared With Me', color: 'neutral' },
+    // { name: 'Trash', color: 'neutral' },
+  ]
+
+  const newProjectButtons = [
+    {
+      title: '+ Blank Project',
+      onClick: createNewProject,
+      color: 'orange',
+    },
+    // {
+    //   title: '+ Project On GitHub',
+    //   onClick: createNewProject,
+    //   color: 'pink',
+    // },
+    // {
+    //   title: '+ Import From GitHub',
+    //   onClick: createNewProject,
+    //   color: 'purple',
+    // },
+    // {
+    //   title: '+ Remix Project',
+    //   onClick: createNewProject,
+    //   color: 'blue',
+    // },
+    // {
+    //   title: '+ Shopify Store',
+    //   onClick: createNewProject,
+    //   color: 'green',
+    // },
+  ] as const
+
+  const [isDarkMode, setIsDarkMode] = useState(false)
+
+  useEffect(() => {
+    const handleColorSchemeChange = (event: {
+      matches: boolean | ((prevState: boolean) => boolean)
+    }) => {
+      setIsDarkMode(event.matches)
+    }
+
+    const mediaQuery = window.matchMedia('(prefers-color-scheme: dark)')
+    setIsDarkMode(mediaQuery.matches)
+    mediaQuery.addListener(handleColorSchemeChange)
+
+    return () => {
+      mediaQuery.removeListener(handleColorSchemeChange)
+    }
+  }, [])
+
+  const logoPic = isDarkMode
+    ? 'url(https://github.com/concrete-utopia/utopia/blob/master/editor/resources/editor/pyramid_dark.png?raw=true)'
```

Comment:
I think it could make sense to copy these two images in the `public/` folder of the Remix app and link to them 🤔